src/debugger/Debugger.js

```diff
@@ -50,21 +50,21 @@ export class DebugServer {
   _realm: Realm;
   _variableManager: VariableManager;
   /* Block until adapter says to run
-  /* runCondition: a function that determines whether the adapter has told
-  /* Prepack to continue running
+  /* ast: the current ast node we are stopped on
   */
-  waitForRun() {
+  waitForRun(ast?: BabelNode) {
```

Comment:
Why is "ast" optional?

Reply:
`waitForRun` is called in the constructor where there is no ast yet.